.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: node_js
+node_js:
+  - stable
+  - 8.9.4
+
+services:
+  - postgresql
+
+cache:
+directories:
+  - "node_modules"
+
+
+before_script:
+  - psql -c 'create database travis_ci_test;' -U postgres
+
+install:
+  - npm install
+script:
+  - npm run test
+
+
+after_script:
+  - npm run coverage
```

Comment:
You will want to run test coverage after all the test has passed. using `after_script` here is not the right custom command to use. see the docs for the right custom command to use.

Reply:
I will check this and implement this feedback